components/playlist/buildflags/buildflags.gni

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import("//extensions/buildflags/buildflags.gni")
+
+declare_args() {
+  enable_playlist = enable_extensions || is_android
```

Comment:
Hmm, I think `is_win || is_mac || is_linux || is_android` would be correct.
As we'll not use extension for playlist, `enable_extensions` is not needed?

Reply:
Yep. Done.